### ISSUE TITLE
YAm 2.8 beta, build from 20.12.12 on AROS. Open window problem.

### ISSUE THREAD

**Issue by jens-maus** [jens-maus/yam#361]
**Originally by _nik-tom@online.no_ on 2012-12-22 08:57:46 +0100**

___
If I click button for new mail or try to open the config window nothing happens.
I ran sashimi and got a message:

Could not create object of Groub.mui
Couldn-t find Group.mui

With YAM 2.7 it works as normal. Is there something new with 2.8 that AROS does
not support, group.mui class ?

**Comment by jens-maus** [comment]
**Originally on 2012-12-22 09:51:29 +0100**

___
To investigate the problem it would be helpful if you could run some tests with the debug version of YAM 2.8-dev. Please download it at the same position like the normal nightly builds. In addition, please start yam from the command-line and use the following command before starting yam:

```
setenv yamdebug stdout gui
```

then catch the debug output when trying to open the config window and upload it here for investigation.

**Comment by jens-maus** [comment]
**Originally by _anonymous_ on 2012-12-22 23:27:13 +0100**

___
**Attachment added:** [`log.txt`](https://yam.ch/attachments/361/log.txt) (22.9 KiB)

**Comment by jens-maus** [comment]
**Originally by _anonymous_ on 2012-12-22 23:40:08 +0100**

___
Added debug. Hope it helps.

**Comment by jens-maus** [comment]
**Originally on 2012-12-22 23:42:22 +0100**

___
Thanks. However, the debug info doesn't contain all the info we need for identifying the problem. Can you please elaborate at which point you tried to open the config window? In addition please try to recreate the log with the following setenv call before starting yam:

```
setenv yamdebug stdout all @all
```

This will generate a huge debug log so be prepared that YAM will get quite slow. Please outline at which point you tried to open the config window and where the error window was presented to you.

**Comment by jens-maus** [comment]
**Originally on 2012-12-22 23:47:52 +0100**

___
BTW: please also create yourself an account here on the webpage so that we don't have to moderate each post and you get directly notified of changes of the tickets.

**Comment by jens-maus** [comment]
**Originally by _nik-tom@online.no_ on 2012-12-23 00:59:50 +0100**

___
Hi.

The last debug you got from me I got using sashimi.

Using the command " setenv yamdebug stdout all @all " works and I got a lot of debug output from shell, but it is not saved to anywhere ?

**Comment by jens-maus** [comment]
**Originally on 2012-12-23 01:04:47 +0100**

___
Well, if you use "stdout" in the setenv call it will be put on the command-line. If you omit it it will be sent to the serial line, thus you can catch it via sashimi. However, you can also add "file:ram:debug.txt" and it will save the debug output to "ram:debug.txt". See http://yam.ch/wiki/FAQ/Debugging for more information on how to create proper debug output using the "yamdebug" environment variable.

**Comment by jens-maus** [comment]
**Originally by _nik-tom@online.no_ on 2012-12-23 11:51:29 +0100**

___
For some strange reason debug.txt is created but file size is 0k so nothing is saved to it.

This is my command line before running YAM.debug

 setenv yamdebug stdout all @all file:ram:debug.txt

**Comment by jens-maus** [comment]
**Originally on 2012-12-23 11:53:12 +0100**

___
Well, you have to remove "stdout" from the command-line if you want to save to a file. Then it should work.

**Comment by jens-maus** [comment]
**Originally by _nik-tom@online.no_ on 2012-12-23 17:54:38 +0100**

___
**Attachment added:** [`debug.txt`](https://yam.ch/attachments/361/debug.txt) (636.8 KiB)

**Comment by jens-maus** [comment]
**Originally by _nik-tom@online.no_ on 2012-12-23 17:55:38 +0100**

___
Hope this one can help you detecting the problem.

**Comment by jens-maus** [comment]
**Originally on 2012-12-23 18:40:44 +0100**

___
Thanks for the log file. However, it is hard to identify the problem from that file. Can you please tell me at which line you tried to open the config window? All I can see is that you seem to have tried to open a write window and that failed also. However, no definite error message is written. Can you please also state which AROS version you are running?

**Comment by jens-maus** [comment]
**Originally by _nik-tom@online.no_ on 2012-12-23 19:29:42 +0100**

___
I'm using latest AROS night build. I also the autour of AspireOS distribution. As said with YAM 2.7 I have no problem at all. The problem with 2.8 is that even if the program is running and seams normal, nothing happen if I try to write mail pushing the new button and if I try to push the config. button AROS freeze. It is little difficult to debug pressing config button since AROS freeze. I could always save the debug to sys: but don't know if it will catch the error ?

**Comment by jens-maus** [comment]
**Originally on 2012-12-23 19:32:47 +0100**

___
Hmm. unfortunatley I don't have a working AROS installation myself. Is there still a VirtualBox version of Icaros or something like that? That would allow me to debug things on my own. Apart from that we really rely on you supplying enough information to debug why YAM 2.8 doesn't seem to work with the latest versions of AROS.

**Comment by jens-maus** [comment]
**Originally by _nik-tom@online.no_ on 2012-12-23 19:37:28 +0100**

___
I have little experience with virtualbox, but know that many run Icaros desktop with it.
If you have a Linux installation running it is quite easy to run AROS hosted. 

Here is a direct link to the AROS, Linux hosted version 

http://netcologne.dl.sourceforge.net/project/aros/nightly/20121222/Binaries/AROS-20121222-linux-i386-system.tar.bz2

**Comment by jens-maus** [comment]
**Originally on 2012-12-23 19:39:19 +0100**

___
I am on OSX here, so I would prefer to have an OSX hosted version somehow (e.g. using virtualbox)

**Comment by jens-maus** [comment]
**Originally on 2012-12-23 19:41:09 +0100**

___
adding Matthias Rustler to this ticket as he is partly our AROS maintainer and has already done quite some changes. Matthias, can you please elaborate if YAM 2.8+ really doesn't work on AROS anymore?

**Comment by jens-maus** [comment]
**Originally on 2013-01-14 23:27:25 +0100**

___
Matthias? Are you listening?

**Comment by jens-maus** [comment]
**Originally on 2013-06-30 21:00:41 +0200**

___
After discussion with thore it seems that the reason for that issue have been caused by a bug in the MUI/zune implementation of AROS. However, in the meantime the issue seem to be fixed and thus the ticket shouldn't be valid anymore.
